### DISCUSSION TITLE
Win: OpenGL

### DISCUSSION THREAD

**Discussion by MaMiKoe** [OpenHantek/OpenHantek6022#271] [Q&A] [answered]
Until now I am using some old Notebooks with the 6022 and "normal" software (e.g. PcScope), which uses the original firmware and the original Hantek-Driver. The integrated video-hardware is Intel 855GME and Intel 915GM and with Vista-video-driver they also work satisfying with Win7. 
When testing OH6022 I realized that the video-hardware does not support OpenGL (not OpenGL 2.1+ and also not OpenGL ES 1.2+, OH6022 does not run even mit "-e"). 
I am really not familiar with this, so I would like to ask whether it is a matter of real hardware-capability and -performance, so that a software-emulation of OpenGL/ES (if possible, I don´t know) would result in a much to bad performance, a  much to slow display. Or whether it is more a "formal" matter and not of hardware-performance.

**Comment by Ho-Ro** [comment, marked as answer]
Sorry to say, but OpenHantek6022 (as well as its ancestor openhantek) are developed under Linux to be able to use the Hantek scope with Linux. Due to using Qt it works *also* under MacOS and Windows and FreeBSD, but especially Windows has to provide some libraries/driver that are not required for the "Win-Way" of doing graphic.
